api/src/main/resources/rdr.yaml

```diff
@@ -0,0 +1,202 @@
+          required: true
+          name: workspaces
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/RdrWorkspace"
+      responses:
+        200:
+          description: successful operation
+
+  /workbench/directory/researchers:
+    post:
+      description: Submits one or more researcher snapshots for storage in the RDR.
+      tags:
+        - Rdr
+      operationId: exportResearcheres
+      consumes:
+        - application/json
+      parameters:
+        - in: body
+          required: true
+          name: researchers
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/RdrResearcher"
+      responses:
+        200:
+          description: successful operation
+
+
+
+definitions:
+  RdrWorkspace:
+    type: object
+    properties:
+      workspaceId:
+        type: integer
+      name:
+        type: string
+      creationTime:
+        type: string
+        format: date-time
+      modifiedTime:
+        type: string
+        format: date-time
+      status:
+        type: string
+        description: Workspace Status
+        enum:
+          - ACTIVE
+          - INACTIVE
+      workspaceUsers:
+        type: array
+        items:
+          $ref: '#/definitions/RdrWorkspaceUser'
+      excludeFromPublicDirectory:
+        type: boolean
+      diseaseFocusedResearch:
+        type: boolean
+      diseaseFocusedResearchName:
+        type: string
+      otherPurposeDetails:
+        type: string
+      methodsDevelopment:
+        type: boolean
+      controlSet:
+        type: boolean
+      ancestry:
+        type: boolean
+      socialBehavioral:
+        type: boolean
+      populationHealth:
+        type: boolean
+      drugDevelopment:
+        type: boolean
+      commercialPurpose:
+        type: boolean
+      educational:
+        type: boolean
+      otherPurpose:
+        type: boolean
+
+  RdrWorkspaceUser:
+    type: object
+    properties:
+      userId:
+        type: integer
+      role:
+        type: string
+        enum:
+          - READER
+          - WRITER
+          - OWNER
+      status:
+        type: string
+        description: Workspace User Status
+        enum:
+          - ACTIVE
+          - INACTIVE
+
+  RdrResearcher:
+    type: object
+    properties:
+      userId:
+        type: integer
+      creationTime:
+        type: string
+        format: date-time
+      modifiedTime:
+        type: string
+        format: date-time
+      givenName:
+        type: string
+      familyName:
+        type: string
+      streetAddress1:
+        type: string
+      streetAddress2:
+        type: string
+      city:
+        type: string
+      state:
+        type: string
+      zipCode:
+        type: string
+      country:
+        type: string
+      ethnicity:
+        type: string
+#      Need to talk to Yu about this to be a list
+      gender:
+#      Need to talk to yu about this to be a list
```

Comment:
Or an object. I'd probably make it an object that contains a list so you have more room to add stuff later.